packages/documentation/src/stories/foundation/layout/container/container.docs.mdx

```diff
@@ -0,0 +1,23 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import * as ContainerStories from './container.stories';
+
+<Meta of={ContainerStories} />
+
+# Container
+
+Containers give resonsive paddign and margins to contents on a webpage. As our Grid is Based on Bootstrap negative Margins are used. Therefore Containers are required to wrap the Grid to prevent unintended side-effects.
```

Comment:
```suggestion
Containers give responsive padding and margins to contents on a webpage. As our grid is based on Bootstrap, negative margins are used. Therefore, containers are required to wrap the grid to prevent unintended side-effects.
```